### ISSUE TITLE
concepts without mappings

### ISSUE THREAD

**Issue by LinguList** [tupian-language-resources/tuled#33]
```
WARNING Missing concept BLUE YELLOW MACAW                                                                                                                                                    
WARNING Missing concept SURUBIM
WARNING Missing concept RED MACAW
WARNING Missing concept HARPY EAGLE
WARNING Entry ID=5438, concept=GNAT / PIUM, language=Mundurukú is empty
WARNING Entry ID=6619, concept=SEND, language=Awetí is empty
WARNING Entry ID=16220, concept=DOOR, language=Mbya is empty
WARNING Missing concept STAY
WARNING Missing concept FLOAT
WARNING Missing concept ENEMY
WARNING Missing concept BE HUNGRY
WARNING Missing concept SHRIMP
WARNING Missing concept BABY CARRIER
WARNING Missing concept EARRING
WARNING Entry ID=22297, concept=MILK, language=Siriono is empty
WARNING Missing concept SCREAM
WARNING Missing concept LIP
```

**Comment by LinguList** [comment]
the problem is that we now use the standardized concept list [Gerard-2021-414](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/Gerardi-2021-414.tsv). This concept list should be our basis for the future, but since the concepts kept changing, we have now again trouble mapping the ones shown above.

If you could indicate to which of the 414 concepts in concepticon they should be assigned, we could update them. 

If you could also list those which cannot be found there, we could add exceptions for the time being, and later publish a new concept list with concepticon. But for now, I want to use the one we already have.

**Comment by LanguageStructure** [comment]
I will take care of these

**Comment by LanguageStructure** [comment]
Should I change (insert) them on GitHub?

**Comment by LinguList** [comment]
no on edictor, please. I do the download and compilation later, once 
this is done.

**Comment by LinguList** [comment]
sorry, I thought you talk about the other problem.

**Comment by LinguList** [comment]
I first want you to carefully check the list which I linked above, which is in concepticon, and see if you modified any wording there, as this has happened a lot in the past.

**Comment by LanguageStructure** [comment]
We have fixed many things already on EDICTOR, and we also have added many
things, including concepts, which have already been assigned a CONCEPTICON
ID.

**Comment by LinguList** [comment]
But if you confirm, that neither of the concepts is now available, also not in this older list: https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/Gerardi-2021-244.tsv

Then please add them here in a tabular form with the concepticon IDs.

**Comment by LanguageStructure** [comment]
I will check it tomorrow evening. SOme of the concepts are not part of
concepticon yet

**Comment by LinguList** [comment]
Because what I do not understand is: Tuled 2021 has 414 concepts, but now, I find only 407 in the list on edictor, but then I find those indicated missing above, so there's an inconsistency between TULED versions which I find surprising. 

It is not tragic, but we would want to know what's going on.

**Comment by LinguList** [comment]
So my suggestion is to clarify above-mentioned concepts, and then potentially make a new concept list, which we'd then not change unless there's a bigger new release, so only concepts listed in this concept list would then go into the website, until one makes a new release with a new authoritative concept list.

**Comment by LanguageStructure** [comment]
The last version we published (0.11) has 420 concepts.
As of now we have 424 on EDICTOR (4 new concepts)

Why we went in the older version from 414 to 407 I also do not understand.
I will meet with Stanislav and try to figure it out

**Comment by LanguageStructure** [comment]
This sounds reasonable
